api/v1beta1/hyperconverged_types.go

```diff
@@ -29,7 +29,10 @@ type HyperConvergedTuningPolicy string
 
 // HyperConvergedAnnotationTuningPolicy defines a static configuration of the kubevirt query per seconds (qps) and burst values
 // through annotation values.
-const HyperConvergedAnnotationTuningPolicy HyperConvergedTuningPolicy = "annotation"
+const (
+	HyperConvergedAnnotationTuningPolicy HyperConvergedTuningPolicy = "annotation"
+	HyperConvergedHighBurstProfile       HyperConvergedTuningPolicy = "HighBurst"
```

Comment:
let's be consistent here. If `annotation` starts with lower cap, can we do the same for `HighBurst`?